### ISSUE TITLE
Bootstrapping using `bin/maxwell-bootstrap` not working

### ISSUE THREAD

**Issue by tankchintan** [zendesk/maxwell#564]
When I try to bootstrap by inserting a row in the database table directly, everything works as expected. But when I try to do the same using the `bin/maxwell-bootstrap` after some stage nothing seems to happen.

The config properties I am providing, is exactly the same that I provide to normal maxwell script, and that works swimmingly well.

```
root:~/maxwell/maxwell-1.7.2# bin/maxwell-bootstrap --config ../config/basic.properties --database db_name --table db_table --where "id > 66890000" --log_level info
18:18:14,764 INFO  MaxwellBootstrapConnectionPool - MaxwellBootstrapConnectionPool: Getting connection (user/password): jdbc:mysql://db-staging.vpc:3306/maxwell
18:18:15,501 INFO  MaxwellBootstrapConnectionPool - MaxwellBootstrapConnectionPool: Created a new connection
18:18:15,503 INFO  MaxwellBootstrapUtility - counting rows
18:18:15,521 INFO  MaxwellBootstrapUtility - inserting bootstrap start row
waiting for bootstrap to start...
waiting for bootstrap to start...
```

Also, is it possible to run both the normal maxwell CDC replicator AND also bootstrap-maxwell script at the same time? (although I was not doing it in this instance).

Another related question, if I have 2 instances of maxwell running, listening to the same source database if I start a bootstrap (by inserting in db myself) which instance of maxwell will pick up the job? Can I decide which one will pick it up?

Basically what I am trying to get at, is, is it possible to standup a maxwell instance do bootstrapping, then throw away the instance after work is done, while at the same time the normal maxwell instance that is running goes about its business?

**Comment by nddeluca** [comment]
I ran into this issue and it was a because I had maxwell shutdown for a while and it fell behind in the binlog.

The bootstrap doesn't start until the replicator reaches the binlog position where the bootstrap record was inserted, so if maxwell is not caught up to the current position, it could take a while for the bootstrap to start.

You can check the position by running
```sql
show master status;
select * from maxwell.positions;
```
and comparing the two.

**Comment by osheroff** [comment]
so I guess it's not clear from the docs, but the `maxwell-bootstrap` script doesn't actually run a copy of maxwell; it's just a controller script that does the table insert, and the main CDC replicator process will pick up the row and bootstrap the table.

**Comment by tankchintan** [comment]
Ah, Ok. I guess that makes sense. I did see that an entry was created in the db. But since I didnt also have a running maxwell instance I didnt see any activity happening. 

In that case might I recommend updating the message from `waiting for bootstrap to start...` to `Queued $database's $table $where for bootstrapping.` And then exit the process?

Also I guess this answers my related question, the bootstrapping request will be picked up by all the maxwell instances whose configurations allow for tables requested.

**Comment by osheroff** [comment]
Oh you know that's interesting.  I'm not sure there's proper multi-client support for bootstrapping.  Will have to look.

Feel free to PR a message change or a documentation clarification if you get time!

**Comment by osheroff** [comment]
I guess I had left this open to remind me to deal with multi-client bootstrapping.  We dealt with it.

**Comment by jianxp** [comment]
how to solve the problem?
